sample_tracking/app/utils.tsx

```diff
@@ -125,9 +126,14 @@ export function getRanHex(size: number): string {
 
 export function initializeAppIfNecessary() {
   try {
-    return getApp();
+    getApp();
   } catch (any) {
-    return initializeApp(firebaseConfig);
+    const app = initializeApp(firebaseConfig);
+    isSupported().then((isSupported: boolean) => {
```

Comment:
how do we know issSupported will only be true for production?
We do not want analytics on river-sky -- only on timberid-prd

Reply:
Added a check to make sure we are in a prod url before adding analytics.